index.html

```diff
@@ -598,7 +598,7 @@ <h5 class="modal-title" id="exampleModalLabel">Privacy Policy</h5>
       });
     </script>
     <!--Start of Tawk.to Script-->
-    <script type="text/javascript">
+    <!-- <script type="text/javascript">
     var Tawk_API=Tawk_API||{}, Tawk_LoadStart=new Date();
     (function(){
     var s1=document.createElement("script"),s0=document.getElementsByTagName("script")[0];
```

Comment:
Instead of commenting it out, I would delete the entire code block including comments like `Start of Tawk,to script`.